mlflow/server/js/src/components/ExperimentViewUtil.js

```diff
@@ -402,10 +405,24 @@ export default class ExperimentViewUtil {
     return mergedRows;
   }
 
+  /**
+   * Given an array of row metadata returned by getRowRenderMetadata, returns an array of run IDs
+   * sorted by display order.
+   */
+  static getRunIdsSortedByDisplayOrder(rowMetadatas) {
+    return rowMetadatas.flatMap(rowMetadata => {
+      if (!rowMetadata.isParent) {
+        return [];
+      }
+      return _.concat([rowMetadata.runId], rowMetadata.childrenIds || []);
```

Comment:
What would happen if we just used all rowMetadata.runId (i.e., why do we have to exclude non-parent rowMetdata?)? Maybe we could add an inline comment regarding this distinction, unless it's obvious.